app/models/user.rb

```diff
@@ -0,0 +1,35 @@
+class User < ApplicationRecord
+  attr_accessor :remember_token
+  before_create :remember_token
```

Comment:
Please, use two different names. In your case, I think that what comes close to the `before_create` is something between `digest` and `new_token`